lib/AST/Decl.cpp

```diff
@@ -2073,6 +2074,25 @@ static AccessLevel getTestableAccess(const ValueDecl *decl) {
   return AccessLevel::Public;
 }
 
+bool ValueDecl::isVisibleExternally(bool isWholeModule) const {
+  AccessLevel access = getEffectiveAccess();
+  SILLinkage linkage;
+  switch (access) {
+    case AccessLevel::Private:
```

Comment:
Formatting nit: ‘case’ should have the same level of indenting as ‘switch’ per standard LLVM/Swift project style.